src/contractTest/java/uk/gov/hmcts/reform/waworkflowapi/provider/WorkflowEvaluateDmnProviderTest.java

```diff
@@ -0,0 +1,88 @@
+package uk.gov.hmcts.reform.waworkflowapi.provider;
+
+import au.com.dius.pact.provider.junit5.PactVerificationContext;
+import au.com.dius.pact.provider.junit5.PactVerificationInvocationContextProvider;
+import au.com.dius.pact.provider.junitsupport.IgnoreNoPactsToVerify;
+import au.com.dius.pact.provider.junitsupport.Provider;
+import au.com.dius.pact.provider.junitsupport.State;
+import au.com.dius.pact.provider.junitsupport.loader.PactBroker;
+import au.com.dius.pact.provider.junitsupport.loader.VersionSelector;
+import au.com.dius.pact.provider.spring.junit5.MockMvcTestTarget;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.springframework.context.annotation.Import;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.waworkflowapi.clients.model.DmnValue;
+import uk.gov.hmcts.reform.waworkflowapi.clients.service.EvaluateDmnService;
+import uk.gov.hmcts.reform.waworkflowapi.clients.service.SendMessageService;
+import uk.gov.hmcts.reform.waworkflowapi.controllers.startworkflow.CreateTaskController;
+import uk.gov.hmcts.reform.waworkflowapi.provider.service.WorkflowProviderTestConfiguration;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+
+@ExtendWith(SpringExtension.class)
+@Provider("wa_workflow_api_evaluate_dmn")
+@PactBroker(
```

Comment:
Can we include the PactFolder to enable local testing and verification? See examples in other provider